.run/PHPStan all.run.xml

```diff
@@ -0,0 +1,10 @@
+<component name="ProjectRunConfigurationManager">
```

Comment:
Could you have a look why this command is failed? When I run phpstan in  the terminal, it works fine.

![image](https://github.com/Sopamo/clustercache/assets/86287030/3170c85c-971a-4d07-bdd3-ca6166bf84e5)

Reply:
I set up a php interpreter that uses docker compose and looks like this:
![image](https://github.com/Sopamo/clustercache/assets/1202596/5c960931-167f-42a7-a547-c24fb43c6a82)
Then it works:
![image](https://github.com/Sopamo/clustercache/assets/1202596/a1058710-510b-43f3-b970-cfde3a603b8d)
The warnings regarding variables that are not set appear, because this project uses laravel sail, which sets environment variables automatically when running docker (compose) commands. PHPStorm doesn't know about them, but you can set env variables manually in the php interpreter settings.
If you don't see any differences to my config, I think we'll have to look at it together. Also make sure to restart all of the docker containers though, I could imagine that something was off on your machine due to the new volume mounts.